File: src/main/resources/archetype-resources/src/main/resources/__artifactId__/skeletons/js/Skeleton__gameName__Player.js
#set( $symbol_pound = '#' )
#set( $symbol_dollar = '$' )
#set( $symbol_escape = '\' )

// ------------------------ Skeleton --------------------------------

load('${artifactId}/wrappers/js/${gameName}Player.js');

(function (exports) {'use strict';

    /**
     * ${gameName} player in Javascript
     *
     * @author ${author} <code>${authorEmail}</code>
     */
    function My${gameName}Player() {
        ${gameName}Player.call(this);
    }

    My${gameName}Player.prototype = Object.create(${gameName}Player.prototype);

    // ---------------------- Your code below ---------------------------

    /**
     * Get the name of the player
     *
     * @returns {string} name of the player
     */
    My${gameName}Player.prototype.getName = function () {
        return '${author}';
    };

    /**
     * Initialize the player
     */
    My${gameName}Player.prototype.init = function () {


    };

    /**
     * Execute player action
     */
    My${gameName}Player.prototype.execute = function () {


    };

    // ------------------------ Skeleton --------------------------------

    exports.Player = My${gameName}Player;
}(this));

File: src/main/resources/archetype-resources/src/main/resources/__artifactId__/skeletons/java/Skeleton__gameName__Player.java
#set( $symbol_pound = '#' )
#set( $symbol_dollar = '$' )
#set( $symbol_escape = '\' )

import ${artifactId}.wrappers.java.${gameName}Player;

import java.util.Random;

/**
 * ${gameName} player in Java
 *
 * @author ${author} <code>${authorEmail}</code>
 */
public class My${gameName}Player extends ${gameName}Player {

    @Override
    public String getName() {
        return "${author}";
    }

    @Override
    public void init() {


    }

    @Override
    public void execute() {


    }
}
